Jenkinsfile

```diff
@@ -228,8 +232,22 @@ try {
 				}
 			}
 
+			bfdEnv = 'test'
+			stage('Deploy base to TEST') {
```

Comment:
Genuinely curious: does everything work correctly here with `bfdEnv` being set outside of a `stage` block/scope? Previously, this variable was set within the `stage`. I know in a Declarative Pipeline, code within a `pipeline` block but _not_ in a `stage` would blow up.

Reply:
I was curious about this, too. There isn't any magic going on here as far as I know, to such an extent that I had wondered if we wouldn't be better served by wrapping the stages in `withEnv` blocks and so on. All that said, I've tested this against `test`, and it works.